packages/api/src/hooks/useAuthentication.ts

```diff
@@ -13,6 +13,8 @@ const useAuthentication = () => {
 
         return {
             ...get_account_status_data.authentication,
+            /** client has been authenticated with IDV photo ID feature */
+            is_authenticated_with_idv_photoid: needs_verification.has('is_authenticated_with_idv_photoid'),
```

Comment:
`is_authenticated_with_idv_photoid` willl be a status in account_status . Shouldnt we get it as `is_authenticated_with_idv_photoid: account_status.has('is_authenticated_with_idv_photoid'),`